taskwiki/vwtask.py

```diff
@@ -165,6 +165,11 @@ def from_line(cls, cache, number):
         if self.parent:
             self.parent.add_dependencies |= set([self])
 
+        # Make task inherit parent task tag and project
+        if self.parent:
+            self.task['tags'].update(self.parent.task['tags'])
+            self.task['project'] = self.parent.task['project']
+
```

Comment:
I'm wondering whether we want to do this for existing tasks or just for new ones... Not sure though.

Anyway, this deserves some tests.

Reply:
Finally found some time for this. As I prefer to only inherit when creating new tasks, I ended up applying this modification: https://github.com/liskin/taskwiki/commit/5740aaa5b2815caf2636f65359171c866ac10052
I'm not sure about `self.task[]` vs `self[]`, the defaults applying block uses the latter, so I use it as well, but I don't use tags very often so without tests I have no idea whether it works correctly or not. I'd help you with those but I'm not sure how soon I'll get to it. Might take a while.

Reply:
I'm not sure how to reply, to be honest. I abandoned nvim and taskwarrior and switched to using emacs and org-mode a month ago.

I don't intend to complete this PR, sorry.

Reply:
Didn't really expect you to reply, mostly just wanted to tell you that I haven't forgotten about this.
If you moved onto something different and don't intend to complete this one, feel free to close this. I'll open a PR from my branch and write the tests later. (Assuming there's a hope @tbabej will become active again, but so far he always did after a few months... :-))